aws_fuzzy_finder/main.py

```diff
@@ -77,6 +86,24 @@ def entrypoint(use_private_ip, key_path, user):
         lib_path
     )
 
+    ssh_command = ENV_SSH_COMMAND_TEMPLATE.format(
+        user=ENV_SSH_USER or user,
+        key=ENV_KEY_PATH or key_path,
+        host=choice(fuzzysearch_bash_command),
+    )
+
+    if vpc:
+        ssh_command += " -t " + ENV_SSH_COMMAND_TEMPLATE.format(
+            user=ENV_VPC_SSH_USER or vpc_user,
+            key=ENV_VPC_KEY_PATH or vpc_key_path,
+            host=choice(fuzzysearch_bash_command),
+        )
+
+    print(ssh_command)
+    subprocess.call(ssh_command, shell=True, executable='/bin/bash')
+
+
+def choice(fuzzysearch_bash_command):
```

Comment:
Cool. I'm glad this is found to be useful. I'll work on get the items fixed
at some point soon.

Kyle

On Sat, May 21, 2016 at 11:41 AM, Piotr Mazurek <notifications@github.com>
wrote:

> In aws_fuzzy_finder/main.py
> <https://github.com/pmazurek/aws-fuzzy-finder/pull/4#discussion_r64136991>
> :
>
> > +        key=ENV_KEY_PATH or key_path,
> > +        host=choice(fuzzysearch_bash_command),
> > +    )
> > +
> > +    if vpc:
> > +        ssh_command += " -t " + ENV_SSH_COMMAND_TEMPLATE.format(
> > +            user=ENV_VPC_SSH_USER or vpc_user,
> > +            key=ENV_VPC_KEY_PATH or vpc_key_path,
> > +            host=choice(fuzzysearch_bash_command),
> > +        )
> > +
> > +    print(ssh_command)
> > +    subprocess.call(ssh_command, shell=True, executable='/bin/bash')
> > +
> > +
> > +def choice(fuzzysearch_bash_command):
>
> I love splitting this to a separate function, and reusing it 👍 It lets
> us use the prompt twice, and is very convenient. 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pmazurek/aws-fuzzy-finder/pull/4/files/89b043593cf244b7dd4922999474eace51bc2a92#r64136991>
>